mandatory/2-function-creation.js

```diff
@@ -15,26 +21,58 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice >= 200) {
+    return totalPrice - (totalPrice * 0.1);
+  } else {
+    return totalPrice - (totalPrice * 0.05);
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let count = 1;
+
+  while (count < limit) {
+    if (count % 2 === 1) {
+    console.log(count);
+    }
+    count = count + 1;
```

Comment:
Nitpick: `count++` is a shorter way of expressing `count = count + 1`

Reply:
Thanks, Michael, noted!